src/games/gi.rs

```diff
@@ -132,12 +155,55 @@ pub fn sniff_inventory(
             {
                 tracing::info!("Found avatar packet");
 
-                for avatar in read_avatars {
-                    if let Some(name) = character_id_map.get(&avatar.avatar_id) {
-                        for guid in avatar.equip_guid_list {
-                            item_guid_character_name_map.insert(guid, name);
+                for avatar in &read_avatars {
+                    if avatar.avatar_type != 1 {
+                        continue;
+                    }
+
+                    let Some(name) = character_id_map.get(&avatar.avatar_id) else {
+                        continue;
+                    };
+
+                    for guid in &avatar.equip_guid_list {
+                        item_guid_character_name_map.insert(*guid, name);
+                    }
+
+                    let Some(level) = avatar.prop_map.get(&4001).map(|prop| prop.val as u32) else {
+                        continue;
+                    };
+
+                    let Some(ascension) = avatar.prop_map.get(&1002).map(|prop| prop.val as u32)
+                    else {
+                        continue;
+                    };
+
+                    let constellation = avatar.talent_id_list.len() as u32;
+
+                    tracing::info!("char {name}: {avatar:#?}");
```

Comment:
Maybe use `trace` instead of `info` to avoid spamming the client?

```suggestion
                    tracing::trace!("char {name}: {avatar:#?}");
```

Reply:
Good call. Done